Controllers/userController.js

```diff
@@ -165,11 +172,40 @@ const logout = async (req, res) => {
 // the app know whether or not a user is logged in.
 const authorized = async (req, res) => {
   try {
-    let userInfo = await redisClient.GET("UserInfo");
+    const { token } = getToken(req);

```

Comment:
Shouldn't this be id? Since you returned an  object if you want to call it token then it'd be
{token:  id}